wremnants/CardTool.py

```diff
@@ -492,7 +585,12 @@ def writeOutput(self, args=None):
         self.writeForProcesses(self.nominalName, processes=self.datagroups.groups.keys(), label=self.nominalName)
         self.loadNominalCard()
         if self.pseudoData and not self.xnorm:
-            self.addPseudodata([x for x in self.datagroups.groups.keys() if x != "Data"])
+            self.addPseudodata([x for x in self.procDict.keys() if x != "Data"],
+                               [x for x in self.procDict.keys() if x != "Data" and not self.pseudoDataProcsRegexp.match(x)])
```

Comment:
So is this pseudoDataProcRegexp used to exclude processes from the pseudo data sum...?

Reply:
Not exactly. The name is actually a bit misleading, it is used to set the processes to get from the alternate datagroups, while the others will be taken from the nominal one (the previous logic was "all or nothing")